app/lib/hooks/useVideoConf/index.tsx

```diff
@@ -45,7 +45,7 @@ export const useVideoConf = (
 				return true;
 			} catch (error: any) {
 				const isAdmin = !!user.roles?.includes('admin');
-				handleErrors(isAdmin, error?.error || 'NOT_CONFIGURED');
+				handleErrors(isAdmin, error?.data?.error || availabilityErrors.NOT_CONFIGURED);
```

Comment:
My suggestion is inside the handleErrors, maybe instead of simple call the `i18n.t(`${key}-body`), i18n.t(`${key}-header`)`, I think you should test whether the error is already translate with i18n.isTranslated, then you show the error. This approach will avoid show to the user `[missing en.{key} translation]`. 

WDYT?